src/components/map/WorldMap.tsx

```diff
@@ -0,0 +1,339 @@
+import { Tooltip, TooltipRefProps } from "react-tooltip";
+import useConfig from "@hooks/useConfig";
+import useGeographies from "@hooks/useGeographies";
+import { GEO_CENTER_POINTS } from "@constants/mapCentres";
+import { GEO_EU_COUNTRIES } from "@constants/mapEUCountries";
+import { TGeography } from "@types";
+import { LinkWithQuery } from "@components/LinkWithQuery";
+import GeographySelect from "./GeographySelect";
+import { ZoomControls } from "./ZoomControls";
+
+const geoUrl = "/data/map/world-countries-50m.json";
+
+type TSvgGeo = {
+  geometry: { type: string; coordinates: TPoint[] };
+  id: string;
+  properties: { name: string };
+  rsmKey: string;
+  svgPath: string;
+  type: string;
+};
+
+type TGeoFamilyCounts = {
+  UNFCCC: number;
+  EXECUTIVE: number;
+  LEGISLATIVE: number;
+};
+
+type TGeoMarkers = {
+  lawsPolicies: number;
+  unfccc: number;
+};
+
+type TGeographyWithCoords = TGeography & { coords: TPoint; familyCounts: TGeoFamilyCounts; markers: TGeoMarkers };
+
+type TGeographiesWithCoords = { [key: string]: TGeographyWithCoords };
+
+const geoStyle = (isActive: boolean) => {
+  return {
+    default: {
+      fill: isActive ? "#1F93FF" : "#dfdfdf",
+      stroke: "#fff",
+      strokeWidth: 0.25,
+      outline: "none",
+    },
+    hover: {
+      fill: "#1F93FF",
+      cursor: "pointer",
+      outline: "none",
+    },
+  };
+};
+
+const markerStyle = {
+  default: {
+    fill: "#E4E7EC",
+    stroke: "#1D2939",
+    strokeWidth: 0.25,
+    outline: "none",
+    cursor: "pointer",
+  },
+  hover: {
+    fill: "#1F93FF",
+    stroke: "#1D2939",
+    strokeWidth: 0.25,
+    outline: "none",
+    cursor: "pointer",
+  },
+  pressed: {
+    fill: "#1F93FF",
+    stroke: "#1D2939",
+    strokeWidth: 0.25,
+    outline: "none",
+    cursor: "pointer",
+  },
+};
+
+const MAX_ZOOM = 20;
+const MIN_ZOOM = 1;
+const maxMarkerSize = 10;
+const minMarkerSize = 1.5;
+
+const calculateMarkerColour = (value: number, min: number, max: number) => {
+  const offset = ((value - min) / (max - min)) * 100;
+  return `hsl(200, 50%, ${100 - offset}%)`;
+};
+
+const GeographyDetail = ({ geo, geographies }: { geo: any; geographies: TGeographiesWithCoords }) => {
+  const geography = Object.values(geographies).find((country) => country.display_value === geo);
+  if (!geography) {
+    return (
+      <>
+        <p>We do not have any information for this area yet. ({geo})</p>
+      </>
+    );
+  }
+  return (
+    <>
+      {geography && (
+        <>
+          <p className="font-bold">{geography.display_value}</p>
+          <p>Laws and policies: {geography.familyCounts?.EXECUTIVE || 0 + geography.familyCounts.LEGISLATIVE || 0}</p>
+          <p>UNFCCC documents: {geography.familyCounts?.UNFCCC || 0}</p>
+          <p>
+            <LinkWithQuery href={`/geographies/${geography.slug}`}>View territory profile</LinkWithQuery>
+          </p>
+        </>
+      )}
+    </>
+  );
+};
+
+// TODO:
```

Comment:
Do we have a ticket for this?

Reply:
Ah - that was my TODO list for the feature, I have done it but forgot to delete the TODO